styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -291,14 +291,41 @@
 }
 
 //Case Study
+$screen-xl: 1440px;
+$screen-xs-plus: 481px;
+
+.caseStudy-banner-image {
+  display: none;
+}
+
+.caseStudy-main-image {
+  background-image: url('../imgs/derek/banners/customer-story-background.jpg');
+  background-position: center;
+  background-size: cover;
+  display: none;
+  height: 450px;
+}
+
+.caseStudy-box {
+  background: $white;
+  display: block;
+  margin-bottom: 1px;
+  padding: 20px 50px 150px 20px;
+  text-decoration: none;
+}
 
 .caseStudy-wrapper {
-  background-image: url('../imgs/derek/banners/caseStudy.jpg');
+  background-image: url('../imgs/derek/banners/customer-story-background.jpg');
```

Comment:
So I know the tendency here is that we already have these classes defined so we should just modify them, however...  There is a chance that some of these CSS selectors are being used elsewhere on the site.  As a result we introduce risk.  

Instead, since we're talking about a completely brand new customer story region, we should be using brand new classes/selectors. 

For example see https://github.com/rackerlabs/zoolander/pull/592/files#diff-ef43fa620c83e18c2fa2e5ea7aea1db7R1

Instead of replacing the existing announcement code, I created all new selectors.  

We could potentially even use the same naming convention here. For example: `.rsHomepage-customerStories-wrapper`

We can then target some of this other css for removal at a later time.